slack-base/src/main/java/com/hubspot/slack/client/models/blocks/messages/RichBlockStyle.java

```diff
@@ -0,0 +1,14 @@
+package com.hubspot.slack.client.models.blocks.messages;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum RichBlockStyle {
```

Comment:
This looks like `ListType` to me. `RichBlockStyle` suggests that this can be applied to any rich block but that isn't the case.

Reply:
Makes sense.